server/configManager.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016 Google Inc.
```

Comment:
I don't know that config needs to be exclusive for the protos, especially since the protos live in version dirs a level down.

In my head, server would be limited to strictly the bits required to get the server itself up (main.go, bits required for handling grpc, some middleware perhaps); the other bits would live elsewhere (maybe in root 'mixer' package when appropriate, etc.).

So, one approach could be:
- `mixer.server.AdapterManager` -> `mixer.adapter.Manager`
- `mixer.server.ConfigManager` -> `mixer.config.Manager`

Anyways, we can resolve this later. Please don't let this hold up the review.